src/Generation/FieldDetails.php

```diff
@@ -260,9 +275,22 @@ public function getOneofDesc()
         return !$this->isOneOf ? null : $this->containingMessage->getOneofDecl()[$this->oneOfIndex];
     }
 
-    public function exampleValue(SourceFileContext $ctx)
-    {
-        if ($this->desc->desc->isRepeated()) {
+    /**
+     * Create an example value based off the type of this field.
+     *
+     * @param SourceFileContext $ctx The context this field is attached to.
+     * @param bool $formatStringWithBrackets Whether or not to format a string
+     *             value as "['VALUE']" or "'value'".
+     * @param bool $ignoredRepeated If the field is repeated, setting this value
+     *             to true will return a value of the singular type (treats it as non-repeated).
```

Comment:
But won't wouldn't this break the sample? `GPBUtil` throws an exception if the provided value isn't an array. We could do something instead like `$wrapRepeated = true` and then wrap the values in an array, e.g.:

```php
$updateMask = [new FieldMask()];
```

Otherwise, static analysis will fail, and samples will be more confusing.

Reply:
This was cleared up offline, should be good.

Reply:
For anyone following along, the reason is that the sample wraps the parameters in an array. This is so sample function arguments can be scalars